jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java

```diff
@@ -1533,6 +1533,14 @@ public void testMITaskProperties() throws Exception {
     }
 
     @Test
+    public void testNoMIOnEmbeddedSubprocess() throws Exception {
+        Definitions definitions = loader.loadProcessFromJson("simpleEmbeddedSubprocess.json");
+        Process process = getRootProcess(definitions);
+        SubProcess subprocess = (SubProcess) getFlowElement(process.getFlowElements(), "abc");
+        assertNull(subprocess.getLoopCharacteristics().getId());
```

Comment:
@tsurdilo I agree that the current form of the assert is sufficient if we assume the current source code of unmarshaller. However, the unmarshaller can be changed in future. And if we use more restrictive assert (the one I have proposed) I think it can help to avoid similar issues (similar to this one I have caused by my last PR). What do you think?